components/footer.tsx

```
@@ -1,8 +1,8 @@
-"use client"
+"use client";
 
-import Link from "next/link"
-import { ThemeToggle } from "@/components/theme-toggle"
-import { FadeIn } from "@/components/animations/fade-in"
+import Link from "next/link";
+import { ThemeToggle } from "@/components/theme-toggle";
```

ESLint

[error] 'ThemeToggle' is defined but never used.

(@typescript-eslint/no-unused-vars)